components/compliance-service/cmd/inspec_runner/inspec_runner.go

```diff
@@ -40,26 +42,31 @@ func main() {
 func changeToUser(username string) error {
 	user, err := user.Lookup(username)
 	if err != nil {
+		logrus.Errorf("inspec_runner unable to look up user %s: %v", username, err)
```

Comment:
Since this function returns an error, let's wrap the errors and just return them as they are already logged by the caller.